lib/segment/src/index/field_index/full_text_index/mmap_inverted_index/mod.rs

```diff
@@ -195,6 +243,9 @@ impl InvertedIndex for MmapInvertedIndex {
             // Only if the index is within bounds of `point_to_tokens_count`, we decrement the active points count.
             self.active_points_count -= 1;
         }
+        if let Some(_point_to_doc) = &mut self.point_to_doc {
+            // Cannot modify a MmapHashMap
```

Comment:
Is it Ok that we don't remove it here?

Reply:
just removed this, we'll integrate with a postinglist